lib/LTP/utils.pm

```diff
@@ -496,6 +496,12 @@ sub install_from_repo {
     # Lock kernel-default to don't pull it as LTP dependency
     zypper_call 'al kernel-default' if get_kernel_flavor eq 'kernel-64kb';
 
+    # Remove packages if they are installed - useful for testing to resolve
+    # conflicts when 'ltp-stable' is installed and 'ltp' to be installed.
+    for my $pkg (qw(ltp ltp-32bit ltp-stable ltp-stable-32bit qa_test_ltp)) {
+        zypper_call("rm $pkg") if (script_run("rpm -q $pkg") == 0);
```

Comment:
I think you can do `zypper rm ltp ltp-32bit ltp-stable ltp-stable-32bit qa_test_ltp`  without loop.